### ISSUE TITLE
[Quest] [Northrend] Chasing Icestorm: Thel'zan's Phylactery

### ISSUE THREAD

**Issue by warcraftgeekygamer** [dalaranwow/dalaran-wow#2533]
Quest: http://www.wowhead.com/quest=12467/chasing-icestorm-thelzans-phylactery
Quest: http://wowwiki.wikia.com/Quest:Chasing_Icestorm:_Thel%27zan%27s_Phylactery
Video: https://www.youtube.com/watch?v=rzY3ztPHtho
NPC: http://www.wowhead.com/npc=27843/wyrmbait
NPC: http://www.wowhead.com/npc=27844/legion-commander-tyralion

You cannot interact (talk to) with the NPC named "Wyrmbait" to begin this quest. You are supposed to accept the quest from Legion Commander Tyralion and then talk to "Wyrmbait". Because you cannot finish this quest, you are unable to complete the Alliance Wrathgate chain quests.

**Comment by warcraftgeekygamer** [comment]
I do not know what happened, but something was different because as of this morning, I was able to complete this quest. I do not know if a hotfix was applied, or if it was something else -- but I was able to complete the quest.

I will say that the script took quite a long time to run. I was able to interact with "Wyrmbait" and waited an extended period of time for him to go and fetch the giant Frostwyrm. After the frostwyrm approached, I aggro'd it and pulled it closer to the NPC's so they could attack.

I'm curious to know why I could not interact with Wyrmbait, but then later on could interact with him. Hmm..
